src/components/Toast/Toast.module.scss

```diff
@@ -0,0 +1,111 @@
+@import "../../styles/global-css-settings";
+@import "../../styles/typography";
+@import "../../styles/themes";
+
+.mondayStyleToast {
```

Comment:
monday prefix it's a problem :) 
But the fact the the structure of the file is more flat with the script actually work here and it's readable.

Reply:
Removed